static/bidder-info/operaads.yaml

```diff
@@ -0,0 +1,15 @@
+maintainer:
+  email: adx@opera.com
+#gvlVendorID: 
```

Comment:
You can remove the `gvlVendorID` entirely if you aren't registered with IAB Europe.